auto-complete.js

```diff
@@ -62,6 +62,44 @@
             });
         }
 
+
+    window.changeText = () => {
```

Comment:
Avoid using global object `window` for this, it's considered bad practice because of the unscoped access, you could easily replace this function in another place and will break this code.

Instead, create a local function
```js
function changeText() { ... }
```

And add it using JS
```js
var buttons = document.getElementsByClassName('autocomplete-boolean-button');
for (var i = 0; i < buttons.length; i++) {
  var button = buttons[i];
  button.addEventListener('click', changeText);
}
```